lib/android/app/src/main/java/com/reactnativenavigation/presentation/OptionsPresenter.java

```diff
@@ -44,29 +50,64 @@ private void applyTopBarOptions(TopBarOptions options) {
         if (options.hidden == NavigationOptions.BooleanOptions.False) {
             showTopBar(options.animateHide);
         }
+        if (options.drawUnder == True) {
```

Comment:
OptionsPresenter is bound to become messy real quick. One way of keeping it clean and readable is to avoid having any kind of logic in it. Ideally, it should act as a delegate and invoke the appropriate style methods of the views.
So `contentView` won't be a simple view, instead it will be a concrete class or an interface declaring `drawBehindTopBar()` and `drawBelowTopBar()`. This change will make the code readable and easier to understand since the user won't need see how the two methods are implemented, to understand what they are doing.
Now, the user needs to read both methods to understand what are these rules that we are adding and removing. It also means that `OptionsPresenter` makes assumptions on the view and knows it's a direct child of a `RelativeLayout`.